src/client/src/ui/status-connection/StatusButtonContainer.tsx

```diff
@@ -0,0 +1,15 @@
+import _ from 'lodash'
```

Comment:
I think you don't need this?

Reply:
Hmm strange that TypeScript didn't warn me - I thought that was a TSLint rule? Perhaps not :D I've removed it, thanks.